### ISSUE TITLE
Dashboard uses your defined leader key instead of hardcoded SPC

### ISSUE THREAD

**Issue by nunocf** [LunarVim/LunarVim#2863]
### Feature Description

Hello!

I think this is a wonderful bit of work that you've developed and I'm having a wonderful time driving it daily.

I've noticed that on startup, the dashboard shown always displays `SPC` as your leader key, irregardless of what it is set to.

Since I don't use SPC as my leader, I would love to see the dashboard reflect at least my chosen leader key, or maybe even the complete command in the button entries, in case users rebind it.

I'll do my best to get a fork with a fix for this over the next few weeks if you find that this is something worth pursuing, what are your thoughts?

**Comment by rebuilt** [comment]
Have you already defined a different leader key?  
`lvim.leader = "space"`

**Comment by nunocf** [comment]
Yes, I apologise, what I intended to do was get the dashboard to display like this:
![image](https://user-images.githubusercontent.com/22882922/181350983-1ef8e4e9-9e63-4a33-a19f-0d09d9e23fae.png)

I was able to do this by setting the following config:
```
lvim.builtin.alpha.dashboard.section.buttons.entries = {
  { lvim.leader .. " f", "  Find File", "<CMD>Telescope find_files<CR>" },
  { lvim.leader .. " n", "  New File", "<CMD>ene!<CR>" },
  { lvim.leader .. " P", "  Recent Projects ", "<CMD>Telescope projects<CR>" },
  { lvim.leader .. " s r", "  Recently Used Files", "<CMD>Telescope oldfiles<CR>" },
  { lvim.leader .. " s t", "  Find Word", "<CMD>Telescope live_grep<CR>" },
  {
    lvim.leader .. " L c",
    "  Configuration",
    "<CMD>edit " .. require("lvim.config"):get_user_config_path() .. " <CR>",
  }

}

```

**Comment by rebuilt** [comment]
Oh, sorry.  I misunderstood your request.  Yes.  Your solution looks good to me.

**Comment by nunocf** [comment]
Great, would it make sense to move this into the [Dashboard](https://github.com/LunarVim/LunarVim/blob/rolling/lua/lvim/core/alpha/dashboard.lua#L54-L63) config for the repo?

Or is the `lvim.leader` property not accessible there?

I think also if I'm making a PR for this, I need to handle the case of the leader being `SPC` and displaying the string `SPC` instead of ` ` which would probably confuse a lot of users.

**Comment by kylo252** [comment]
fixed in #3021
